docs/tables/cache_behaviour_lambda_function_associations.md

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
I am pretty sure this file shouldn't exist, if it does (find by table name) than that table should begin with aws_ prefix